docs/5.plugins/3.plugin-nlu.mdx

```diff
@@ -25,10 +25,10 @@ Alternatively, if you want to take a closer look of how to make it work from scr
 
 ### **Installing the plugin**
 
-The first step is to install the module with the following command in your bot's project:
+The first step is to install the following modules with the following command in your bot's project:
```

Comment:
not an expert in JS, but I think that a module is a file, and npm installs packages not modules

Reply:
sure,fixed, and yes you are an expert! 😜